app/database/book.py

```diff
@@ -8,7 +8,7 @@ class Book(SQLModel, table=True):
     isbn: int = Field(max_digits=13, unique=True, nullable=False)
     title: str = Field(nullable=False)
     creators: str = Field(nullable=False)
-    copyright_date: date = Field(nullable=False)
+    copyright_date: str = Field(nullable=False)
```

Comment:
Seeing as this is the real `Book` and not `Incomplete` this should probably stay as a `date` unless it's really impossible to parse the incoming `string` to a `date`.

Reply:
I don't agree that it should be a date because we're dealing with several different formats and would cause more headaches and really offer no benefit to enforce the date type. The actual use case is unlikely to be sorting by date, and performance is already not a concern

Reply:
I agree with Cesar that this should be enforce as a date instead of raw `str` mostly for data integrity sake. I don't think the different format will be a problem since they will all be the same when converted to a `date` object.